core/LSU.cpp

```diff
@@ -482,9 +505,15 @@ namespace olympia
                 continue;
             }
 
-            // Pick winner
-            if (!(*win_ptr_iter)->isReady() || (*iter)->winArb(*win_ptr_iter)) {
-                win_ptr_iter = iter;
+            if (stall_pipeline_on_miss_) {
+                if((*iter)->getPriority() == LoadStoreInstInfo::IssuePriority::NEW_DISP) {
+                    return *iter;
+                }
```

Comment:
Perhaps you can point me to the docs for this, but if we are stalling the pipeline on a miss, shouldn't we check to see if the pipeline _is stalled_ before picking new op?  Or are you **only** looking for new ops _because_ of the stalling scenario?

Reply:
@knute-sifive This PR only implements stall on miss, a new instruction is issued only when an instruction exits the pipeline.This is why I am look for instruction which are ready and newly dispatched.